src/software/ai/hl/stp/play/halt_play.cpp

```diff
@@ -23,12 +23,12 @@ bool HaltPlay::invariantHolds(const World &world) const
 void HaltPlay::getNextTactics(TacticCoroutine::push_type &yield)
 {
     // Create Stop Tactics that will loop forever
-    auto stop_tactic_1 = std::make_shared<StopTactic>(false, true);
-    auto stop_tactic_2 = std::make_shared<StopTactic>(false, true);
-    auto stop_tactic_3 = std::make_shared<StopTactic>(false, true);
-    auto stop_tactic_4 = std::make_shared<StopTactic>(false, true);
-    auto stop_tactic_5 = std::make_shared<StopTactic>(false, true);
-    auto stop_tactic_6 = std::make_shared<StopTactic>(false, true);
+    auto stop_tactic_1 = std::make_shared<StopTactic>(false);
+    auto stop_tactic_2 = std::make_shared<StopTactic>(false);
+    auto stop_tactic_3 = std::make_shared<StopTactic>(false);
+    auto stop_tactic_4 = std::make_shared<StopTactic>(false);
+    auto stop_tactic_5 = std::make_shared<StopTactic>(false);
+    auto stop_tactic_6 = std::make_shared<StopTactic>(false);
```

Comment:
I think `loop_forever` should be `true` for all of these?

Reply:
Yes it should. Missed this the first time.

Reply:
Don't know how to link the specific section of code in reply so I commented separately. The boolean is `coast` rather than `loop_forever`. User don't have access to `loop_forever` after change